cellprofiler/modules/rescaleintensity.py

```diff
@@ -439,11 +467,25 @@ def rescale(self, image, in_range, out_range=(0.0, 1.0)):
 
     def stretch(self, input_image):
         data = input_image.pixel_data
-
         mask = input_image.mask
 
-        in_range = (min(data[mask]), max(data[mask]))
+        if len(data.shape) == len(mask.shape) + 1:
```

Comment:
See if you can replace with ndim

Reply:
https://github.com/CellProfiler/CellProfiler/blob/master/cellprofiler/image.py#L105-L106